awscli/compat.py

```diff
@@ -71,6 +71,8 @@
 imap = map
 raw_input = input
 
+OUTPUT_ENCODING_ENV_VAR = 'AWS_CLI_OUTPUT_ENCODING'
```

Comment:
I was on the fence on whether to introduce a new setting as opposed to reusing `AWS_CLI_FILE_ENCODING`, since it's largely the same intent. But the _"file"_ in that name, and the fact that this is new behavior, lead me to a new setting. But open to consolidating if others feel strongly.